packages/app-project/src/shared/components/CollectionsModal/components/CreateCollection/CreateCollection.spec.js

```diff
@@ -12,7 +12,7 @@ describe('CollectionsModal > Component > CreateCollection', function () {
   const onSubmit = sinon.stub()
 
   before(function () {
-    // use mount because we're t4esting a component that uses refs
+    // use mount because we're testing a component that uses refs
```

Comment:
If you change the component to use hooks, this test might need to use `render` instead of `mount`. Or be rewritten with React Testing Library.